components/event-block/EventBlockRelatedStories.vue

```diff
@@ -0,0 +1,190 @@
+<template functional>
+  <div>
+    <img
+      v-if="props.showWaveMarker"
+      src="/event-title-wave.svg"
+      width="111"
+      height="35"
+      alt=""
+      class="event-block-related-stories__icon"
+    >
```

Comment:
Do we also show this wave here? Can't find it in the design.

Reply:
No, it's not in the design, but I'd like to give them the option for a future event.